### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ZackDayn**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -375,6 +375,27 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
           </div>
         </div>
       </div>
+
```
Comment (ShehrozIrfan) [inline index.html:378]: Remove this extra line.

File: index.html [hunk 2]
```diff
@@ -375,6 +375,27 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
...
+            <div class="profile-github pt-2">
+              <p>
+                <a href="https://github.com/ZackDayn" target="_blank"><i
+                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
```
Comment (ShehrozIrfan) [inline index.html:398]: Remove this extra line.

**Comment by ShehrozIrfan** [comment]
@ZackDayn 
Good work. Just remove the extra lines.

**Comment by ZackDayn** [comment]
updated

**Comment by ShehrozIrfan** [comment]
@ZackDayn 
Great!
You can follow me and star this repository for further updates.
Have fun!

**Comment by ShehrozIrfan** [comment]
Hey there!
If your contribution is excluded for this repository, then please visit the repository [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks/) for contribution to hacktoberfest. It contains a plenty of beginner's friendly issues accepted for hacktoberfest.

Have fun!

**Pull request merged**